recipe/meta.yaml

```diff
@@ -3,21 +3,22 @@
 {% set sha256 = "1d19f4cd186eec3dfb363571e3fe2e6d3377386ead6febc6ad45402f0634d2a6" %}
 
 package:
-  name: {{ name|lower }}
+  name: verilator-packages
   version: {{ version }}
 
 source:
   url: https://github.com/{{ name }}/{{ name }}/archive/v{{ version }}.tar.gz
   sha256: {{ sha256 }}
   patches:
-      - patches/0001-treat-DEFENV_VERILATOR_ROOT-as-c_str-for-conda-reloc.patch
+    - patches/0001-treat-DEFENV_VERILATOR_ROOT-as-c_str-for-conda-reloc.patch
```

Comment:
Might want to add a comment about this patch being dropped on the next release with a pointer to the upstream PR that was merged today.

Reply:
```suggestion
    - patches/0001-treat-DEFENV_VERILATOR_ROOT-as-c_str-for-conda-reloc.patch  # remove next bump. see https://github.com/verilator/verilator/pull/4044 
```